### ISSUE TITLE
nucleus.base does not indicate if it should be absolute or not

### ISSUE THREAD

**Issue by Fishrock123** [creationix/nucleus#29]
I assume this should be an absolute path, but it is not specified and no example is given yet.

**Comment by creationix** [comment]
Correct, I've not specified it fully yet.  Currently it's just implementation specific and informational (intentionally undefined), but we can probably standardize it as an absolute path to the base if it's always a real path.

I do have grand plans of eventually implementing nucleus in the browser on top of a paravirtual FS and TCP stack. But I suppose that even in that paravirtual filesystem, there can still be absolute paths.

We can keep it undefined for now till someone has a need to depend on it, or we can define what we thing will be useful and support it forever.
